devops/containers/ubuntu2004_intel_drivers.Dockerfile

```diff
@@ -15,14 +16,17 @@ RUN apt update && apt install -yqq wget
 
 COPY scripts/get_release.py /
 
-# Install IGC and NEO
+# Install IGC, CM and NEO
 RUN python3 /get_release.py intel/intel-graphics-compiler $igc_tag \
   | grep ".*deb" \
   | wget -qi - && \
   python3 /get_release.py intel/compute-runtime $compute_runtime_tag \
   | grep -E ".*((deb)|(sum))" \
   | wget -qi - && \
   sha256sum -c *.sum &&\
+  python3 /get_release.py intel/cm-compiler $cm_tag \
```

Comment:
https://intel.github.io/llvm-docs/GetStartedGuide.html - I don't see cm-compiler in the list of prerequisites in our documentation.
@dongkyunahn-intel, @kbobrovs, is cm-compiler required? If so, please, update the documentation.

Reply:
We have a test in LLVM Test Suite, that explicitly requires CM: https://github.com/intel/llvm-test-suite/blob/74e49b34513f3a24c939be3b8aada17aa83b5d53/SYCL/OnlineCompiler/online_compiler_common.hpp#L104-L105

Reply:
I agree that cm-compiler might be needed to test all DPC++ features, but tests should not fail if it's not installed on the system.
LIT config must have corresponding feature and skip the test if dependencies are not installed. The same way it's done for OpenCL runtime in https://github.com/intel/llvm-test-suite/blob/74e49b34513f3a24c939be3b8aada17aa83b5d53/SYCL/OnlineCompiler/online_compiler_OpenCL.cpp

Reply:
Fair point, added a new feature to llvm-test-suite here https://github.com/intel/llvm-test-suite/pull/611

Reply:
@bader, the only DPC++ feature CM compiler is required for is online compilation from CM source. CM compilation should be available in driver/ocloc. Maybe something changed since this feature was committed.

```
  -options <options>            Optional OpenCL C compilation options
                                as defined by OpenCL specification.
                                Special options for Vector Compute:
                                -vc-codegen <vc options> compile from SPIRV
                                -cmc <cm-options> compile from CM sources

```

Reply:
@v-klochkov, can you please comment?

Reply:
@kbobrovs, note that here we install https://github.com/intel/cm-compiler. Are you saying it should not be needed?

Reply:
> @v-klochkov, can you please comment?

ocloc calls corresponding CM dynamic library if -cmc is passed. So, CM compiler is needed for that LIT test verifying online_compiler feature.
I also agree that the test should not fail if CM is not installed and the patch https://github.com/intel/llvm-test-suite/pull/611/files is a good move. I added my approval there as well - it is ready to be merged now.

Reply:
@bader, I had an impression that driver + ocloc encompasses needed headers and compiler itself and that there is no dependence on stand-alone CM compiler. @v-klochkov - is this not correct?

Reply:
> @bader, I had an impression that driver + ocloc encompasses needed headers and compiler itself and that there is no dependence on stand-alone CM compiler. @v-klochkov - is this not correct?

as soon as you give -cmc to libocloc, it starts requiring another lib*cm* lib doing the compilation for CM.

Reply:
It sounds like driver+ocloc is not enough and there is a dependency on https://github.com/intel/cm-compiler to compile CM sources.

Reply:
Right. I checked recent distros and I don't see the libcm* there anymore.